srcpkgs/kicad-i18n/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'kicad-i18n'
 pkgname=kicad-i18n
-version=5.1.7
+version=5.1.8
 revision=1
 _commit=d24af2da8cab4ce1081c401909a4a880514e5549
```

Comment:
You didn't update the commit. Remember to test build all packages locally :)

Reply:
Looks like I fucked up. Of course I tested the build locally first, I ran `./xbps-src pkg -j 32 kicad-{doc,footprints,i18n,...}`. I thought this would build all of them. Later I tried installing the generated packages and noticed that `kicad-library` wasn't built, so without looking I though I probably forgot it and built it manually again. This now built almost all of them except this one, because library does not depend on i18n :D

I also noticed that the i18n template has the wrong sources and actually builds kicad itself. WTF. I will fix that as well.

Reply:
Sorted things out.